index.md

```diff
@@ -105,16 +128,26 @@ You can also choose to install them using the <a href="https://docs.python-guide
 sudo apt install linux-perf
 ```
 
-### Other
+**NOTE:** For Perf specifically, you need to run the tool using `sudo`, since Perf requires root access.
+
+### Installing from source
+
+It is also possible to download the `.zip` or `.tar.gz` file directly from the <a href="https://github.com/fernivy/fernivy/releases/latest" target="blank">latest release on GitHub</a>.
 
-Check out the <a href="https://github.com/fernivy/fernivy" target="_blank">repository</a>.
+This can be done both for MacOS and Linux, but the dependency on Python 3 and PowerLog/`perf` remains the same. After downloading and unpacking the source, one can generate the necessary scripts using `make generate`. The correct scripts for running `fernivy` from source can then be found in either `powerlog/package` or `perf/package`.
```

Comment:
```suggestion
This can be done both for MacOS and Linux, but the dependency on Python 3 and PowerLog/`perf` remains the same. After downloading and unpacking the source, one can generate the necessary scripts using `make`. The correct scripts for running `fernivy` from source can then be found in either `powerlog/package` or `perf/package`. 
```

(The default of `make` will call generate :wink: )

Reply:
oh nice, done